src/main/java/mekanism/common/integration/multipart/MultipartMekanism.java

```diff
@@ -162,4 +165,23 @@ public static boolean hasConnectionWith(TileEntity tile, EnumFacing side)
 		
 		return Collections.emptyList();
 	}
+	
+	public static boolean isMultiPartContainer(TileEntity entity)
+	{
+		return entity instanceof IMultipartContainer;
+	}
+	
+	public static Collection<TileEntity> getTileEntities(TileEntity container)
+	{
+		if(container instanceof IMultipartContainer)
+		{
+			return ((IMultipartContainer)container).getParts().values().stream()
+					.map(partInfo -> partInfo.getPart())
+					.filter(part -> part instanceof MultipartTile)
```

Comment:
Either change the method name to `getMekanismTileEntities` or remove the filter. `IPartInfo` has a `getTile` method.

Personally I'm not really feeling the `stream` usage; it's very unclear what things are returning and therefore what the block is doing. It also seems to require 3-4 more loops than would be necessary for a plain for loop.  Not great for performance.